javascript/javascript2/week1/individual-number.js

```diff
@@ -0,0 +1,29 @@
+const danishString = ["Jeg har en blå bil"];
+const danishStr1 = danishString.toString();
+
+const danishString2 = ["Blå grød med røde bær"];
+const danishStr2 = danishString2.toString();
+
+function char_count(danishStr) {
+  let res = danishStr.toLowerCase();
+  let danishWords = { æ: 0, ø: 0, å: 0, count: 0 };
+  let letter_Count1 = 0;
+  let letter_Count2 = 0;
+  let letter_Count3 = 0;
+
+  for (let i = 0; i < danishStr.length; i++) {
+    if (danishStr.charAt(i) == "å") {
+      letter_Count1 += 1;
```

Comment:
Try doing something like this when using the object. **Same** for the rest of the  **if** cases. and remove the variables **letter_Count1 , letter_Count2 , letter_Count3**
```suggestion
    danishWords.å  += 1;
```